packages/amplify-cli/src/amplify-service-helper.ts

```diff
@@ -22,6 +25,16 @@ export function constructInputParams(context: $TSContext) {
   return inputParams;
 }
 
-export async function postPullCodeGenCheck(context: $TSContext) {
-  context.print.info('');
-}
+export const postPullCodegen = async (context: $TSContext) => {
+  if (!!context?.exeInfo?.inputParams?.['no-codegen']) {
```

Comment:
why do we need the no-codegen param?

Reply:
per the spec, --no-codegen is supposed to maintain the current behavior